### PR TITLE
Update objects to match BigQuery schema

### PR DESCRIPTION
**Pull request by Corentin-pro**
Fix #708

### CODE REVIEW

File: ethereumetl/jobs/export_all_common.py [hunk 1]
```diff
@@ -144,8 +144,7 @@ def export_all_common(partitions, output_dir, postgres_connection_string, provid
                     'receipt': insert(RECEIPTS),
                     'token': insert(TOKENS),
                 },
-                converters=[ListJoinItemConverter('topics', ','),
-                            ListJoinItemConverter('function_sighashes', ','),
+                converters=[ListJoinItemConverter('function_sighashes', ','),
                             Numeric38Converter('token_transfer', 'value')]
```
Comment (phillipleblanc) [inline ethereumetl/jobs/export_all_common.py]: Do we still need the Numeric38 converter here?

Comment (Corentin-pro) [reply ethereumetl/jobs/export_all_common.py]: Indeed this make no difference in postgre, I didn't change what was not needed (difficulty attributes in block also work even changing the table type)

Comment (Corentin-pro) [reply ethereumetl/jobs/export_all_common.py]: Removing the converter

File: ethereumetl/streaming/postgres_tables.py [hunk 2]
```diff
@@ -39,8 +39,8 @@
     Column('state_root', String),
     Column('receipts_root', String),
     Column('miner', String),
-    Column('difficulty', Numeric(38)),
-    Column('total_difficulty', Numeric(38)),
+    Column('difficulty', String),
+    Column('total_difficulty', String),
```
Comment (phillipleblanc) [inline ethereumetl/streaming/postgres_tables.py]: I think this change is unnecessary

Comment (phillipleblanc) [reply ethereumetl/streaming/postgres_tables.py]: It should actually be Numeric(38, 9)

Comment (Corentin-pro) [reply ethereumetl/streaming/postgres_tables.py]: Yes, I thought we changed the other way around but the data should be Numeric, fixed.

**Review by lukekim — APPROVED** [review]

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**